### ISSUE TITLE
render_trajectory.py bug

### ISSUE THREAD

**Issue by Davidyao99** [Isabella98Liu/DG-Mesh#20]
There seems to be a bug in the mesh_shape_renderer function call in render_trajectory.py file [here](https://github.com/Isabella98Liu/DG-Mesh/blob/754f42c67510a65261a878ae447a6280a7dfa6a0/dgmesh/render_trajectory.py#L141)

It does not seem to match the function parameters [here](https://github.com/Isabella98Liu/DG-Mesh/blob/754f42c67510a65261a878ae447a6280a7dfa6a0/dgmesh/utils/renderer.py#L236).

It throws a parameter unkown for paramters mask, orig_img and bg_color.

**Comment by MichaelWangGo** [comment]
@Davidyao99 yeah I got the same error. Just removing additional parameters is ok.
`mesh_image_shape = mesh_shape_renderer(verts, faces, render_cam)`
